internals/cli/cli.go

```diff
@@ -258,7 +255,7 @@ func (e *exitStatus) Error() string {
 	return fmt.Sprintf("internal error: exitStatus{%d} being handled as normal error", e.code)
 }
 
-func Run() error {
+func Run(config *client.Config) error {
```

Comment:
Maybe name this `clientConfig` to be clear (eg: that it's not daemon or other config).